rgw/v2/tests/aws/test_delete_version_id_null.py

```diff
@@ -63,84 +64,116 @@ def test_exec(config, ssh_con):
-
-        sync_status(ssh_con=remote_site_ssh_con)
-
-        version_list = aws_reusable.list_object_versions(
-            cli_aws, bucket_name, local_endpoint
-        )
-        log.info(
-            f"versions of objects for the bucket {bucket_name} from local site: {current_zone_name} is {version_list}"
-        )
-
-        version_list = aws_reusable.list_object_versions(
-            cli_aws, bucket_name, remote_endpoint
-        )
-        log.info(
-            f"versions of objects for the bucket {bucket_name} from remote site:{remote_zone_name} is {version_list}"
-        )
-
-        # Deleting object with version id null from local site
-        aws_reusable.delete_object(
-            cli_aws, bucket_name, object_name, local_endpoint, versionid="null"
-        )
-        sync_status(ssh_con=remote_site_ssh_con)
-
-        # Verifying object with version id null is deleted from both sites:local and remote
-        aws_reusable.verify_object_with_version_id_null(
-            cli_aws, bucket_name, object_name, local_endpoint, created=False
-        )
-        aws_reusable.verify_object_with_version_id_null(
-            cli_aws, bucket_name, object_name, remote_endpoint, created=False
-        )
-
-        s3_reusable.remove_user(user)
+    if config.test_ops.get("delete_with_verid", False):
+        log.info("Validate Object version deletion with version-id")
+        endpoint = aws_reusable.get_endpoint(ssh_con, ssl=config.ssl)
+        for user in user_info:
+            user_name = user["user_id"]
+            log.info(user_name)
+            cli_aws = AWS(ssl=config.ssl)
+            aws_auth.do_auth_aws(user)
+            bucket_name = "testbkt-1"
+            aws_reusable.create_bucket(cli_aws, bucket_name, endpoint)
+            log.info(f"Bucket {bucket_name} created")
+            aws_reusable.put_get_bucket_versioning(cli_aws, bucket_name, endpoint)
+            object_name = "hello.txt"
+            utils.exec_shell_cmd(f"fallocate -l 12m {object_name}")
+            aws_reusable.put_object(cli_aws, bucket_name, object_name, endpoint)
+            log.info(f"upload another version of objcet {object_name}")
+            aws_reusable.put_object(cli_aws, bucket_name, object_name, endpoint)
+            obj_versions = json.loads(
+                aws_reusable.list_object_versions(cli_aws, bucket_name, endpoint)
+            )
+            version_id = obj_versions["Versions"][1]["VersionId"]
+            log.info(f"version_id is : {version_id}")
+            out = aws_reusable.delete_object(
+                cli_aws, bucket_name, object_name, endpoint, versionid=version_id
+            )
+            if "Could not connect to the endpoint URL" in out:
+                raise AssertionError(
+                    "RGW crash seen while deleting object version with version id"
+                )
+            s3_reusable.remove_user(user)
+    elif config.test_ops.get("delete_version_id_null", False):
```

Comment:
@ckulal , can we just use if statement and not elif

Reply:
done